internal/http/http_get.go

```diff
@@ -0,0 +1,48 @@
+package http
+
+import (
+	"errors"
+	"io/ioutil"
+	"net/http"
+
+	"go.riyazali.net/sqlite"
+)
+
+type http_get struct{}
```

Comment:
let's just name this struct `get` since the `http` is implied by the package name. Other method types `POST`, `PUT`, `DELETE` etc can be similar files in the same directory when we get to them